### DISCUSSION TITLE
Vocês já se mudaram para outro estado por causa de emprego?

### DISCUSSION THREAD

**Discussion by camillodev** [frontendbr/forum#444] [Discussão]
Eu estou pensando em me mudar para São Paulo por que o mercado é muito mais forte ai do que no RJ e a comunidade principalmente. Só que não sei se tento arrumar um emprego daqui primeiramemte, e acho dificil as empresas quererem alguem do RJ junior, se eu tento arrumar um home office, ou começo a fazer freelas pra ir, ou se é melhor juntar uma grana pra ficar uns 6 meses em sao paulo e tentar arrumar emprego durante esse tempo. 

Se vocês que se mudaram puderem falar um pouco como foi o processo de vocês ajudaria muito. E quem puder falar do mercado de SP também seria otimo. Tambem penso em Floripa e Porto Alegre. 

Um pouco de mim: Tenho 22 anos, trabalhei sempre como .Net Fullstack e agora to migrando para Front End, estudando fortemente nos ultimos 6 meses.

**Comment by waldemirflj** [comment]
Opa! Cara, aconselho você a juntar uma grana. Eu fiquei 7 meses em Florianópolis/Santa Catarina e a única verba que eu tinha era a grana que eu recebia. Passei por alguns problemas e se eu tivesse uma grana extra talvez eu ainda estivesse lá.

**Comment by RatoX** [comment]
Eai,


Cara sou da Bahia em 2013 me mudei pra Recife pra trampar na ThoughtWorks, eles pagaram todas as minhas despesas de viagem, depois fui pra São Paulo também pela ThoughtWorks. Tive alguns problemas por não ter uma grana guardada mas nada que fez mal. 

Uma dica que eu dou: não perca oportunidades.

**Comment by camillodev** [comment]
A questão é que ainda não tenho nenhuma oportunidade. A questão é realmente essa, se vou com a cara e coragem e uma grana pra ficar uns 6 meses  pra sao paulo ou se procuro uma vaga que aceite pessoas de outro estado primeiramente.

**Comment by hugocarreira** [comment]
Opa, então @camillodev sem querer parecer mal ou algo do tipo, mas o mercado está bem movimentado no RJ, moro aqui e vejo bastante vaga boa rolando, com remuneração boa e boas oportunidades.

Não me interprete mal, mas tenho você no facebook, creio que por causa do Encontro de Front RJ que você tava organizando, e acompanho algumas postagens sua quando aparece no meu feed, e vejo você reclamando muito de vagas/empresas, você já parou para pensar que o erro pode não ser o mercado ou as empresas? 

Enfim, não me interprete como rude, por favor. Só estou dizendo o que eu vivi também.

E sobre sua ideia de se mudar e tal, cara, RJ é do lado de SP, você consegue passagem de avião ponte-aérea por menos de 100 reais, o que eu te aconselho a fazer é começar a aplicar pra vagas em SP e vai pra lá só pra entrevista... Creio que vai ser bem melhor assim, você não precisa ficar passando perrengue / juntar grana pra isso.

**Comment by adrianoavelino** [comment]
A Lambda3 tem um podcast com esse assunto:
https://blog.lambda3.com.br/2016/12/podcast-23-trabalhando-em-outro-estado/

**Comment by camillodev** [comment]
então @hugocarreira,agradeço pelo seu feedback, de verdade, mas eu so reclamei de dois processos seletivos que participei que foram bem desorganizados. Em um deles eu perdi o dia inteiro, foi bem desorganizado para depois me responderem que não querem mais um javascript developer e sim outra coisa. Eu acho essa desorganização da empresa bem ruim, por que sinceramente disperdissou o tempo de muita gente.  E os que eu peguei contato concordaram. 

E eu realmente não vejo o mercado de front tão aquecido, principalmente por que eu vejo em media 3 a 5 vagas por semanas em geral, e normalmente as vagas são ou pra estagio e nao rola pra mim pq não faço mais faculdade ou para pleno e senior. Então é frequente eu ouvir que eu sou muito qualificado para uma vaga de estagio, e pouco experiente para uma de pleno que é completamente verdade. 
Eu estou estudando para tentar compensar isso,

Enquanto em sp esse numero é muito maior, e aqui também era maior antigamente e eu sei por que não é hoje. O mercado de fullstack e back end aqui é bem mais forte. Em SP ocorrem eventos de Front com muita frequencia, aqui já é mais dificil. E tem bastante vagas de junior também, então acredito que lá eu tenha mais oportunidades de crescimento. 

Sobre ir para SP a cada entrevista é bem inviavel, pq é em media 150 reais a passagem e muitas dos processos seletivos são varias etapas,  e acredito que pela quantidade de vagas que tem em sp eu faria bastante entrevistas então sairia bem caro no total. Se eu fizer entrevistas ou etapas seria  600 reais, e devo conseguir dividir um quarto/republica o que for por esse preço.

**Comment by RatoX** [comment]
@camillodev Uma dica boa cara é tu achar a empresa que tu quer trampar e começar a se preparar pro processo seletivo dela.

Aqui em SP tem um monte de empresa: ThoughtWorks, Vila Real, NuBank, Banco Original e entre um monte de outras com bons ambientes.

**Comment by camillodev** [comment]
Acho que vou fazer uma lista delas e começar a participar dos processos. 
Mas antes vou tentar começar a trabalhar com freelancer ou remoto para poder me mudar logo e aumentar as chances de trabalhar em SP.

**Comment by luizangelomedeiros** [comment]
@camillodev Passei por algo parecido recentemente.
Sai de Passo Fundo no interior do RS e fui para Floripa.

Estudei essa mudança por um bom tempo, juntei dinheiro e comecei a selecionar as empresas em que eu gostaria de trabalhar. Pois , ao meu ver, não adianta tu achar uma vaga em que tu preencha os requisitos e a empresa "não ter pensamentos/objetivos" parecidos com o seu.

Quando decidi em definitivo que gostaria de mudar, criei no google keep três notas chamadas São Paulo, Curitiba e Florianópolis, que eram as cidades que eu tinha interesse. Nessas notas eu fui adicionando empresas que eu tinha como referencia e colocando os pros e contras de cada cidade.

Após me decidir por Floripa, enviei alguns currículos e participei de alguns processos seletivos até achar a vaga que procurava. Tudo isso eu fiz em minha cidade e depois pedi 30 dias para me organizar nessa mudança, que não é simples por sinal.

Minhas dicas seriam:

- Procura alguns reviews de empresas no [LoveMondays](https://www.lovemondays.com.br/) ou [GlassDoor](https://www.glassdoor.com/)

- Participa de alguns processos seletivos em plataformas como [GeekHunter](https://www.geekhunter.com.br/) e [Contratado.me](http://www.contratado.me/)

- Tenta ver como é o mercado na cidade que escolher. Por exemplo, da uma olhada nos [Meetups](https://www.meetup.com/pt-BR/) , entra em comunidades do Facebook de Front-end da respectiva cidade e da uma pesquisada nas vagas [no próprio forum do FrontendBR](https://github.com/frontendbr/vagas/issues)

- Vai com calma, sei que é complicado, o cara fica ansioso para conseguir o quanto antes, mas vai analisando as propostas, colocando os prós e contras que uma hora aparece uma oportunidade ideal para ti. 

Acho que era mais ou menos isso, mals o textão HAHAHHAHA
boa sorte

**Comment by camillodev** [comment]
@luizangelomedeiros  brigadão pelo textão hahaha. Nossa ajudou muito. 

Sobre "não adianta tu achar uma vaga em que tu preencha os requisitos e a empresa "não ter pensamentos/objetivos" parecidos com o seu." eu to aprendendo isso agora, mas finalmente aprendi. 
Eu to começando esse processo que você falou mesmo, juntar grana, pesquisar mais da cidade, das empresas, cultura da empresa e se candidatar com calma. A ansiedade bate, mas é melhor fazer com calma do que cair numa furada.

To planejando ir daqui a 6 meses por isso, pra juntar uma boa grana e me preparar para todos os processos seletivos.  Brigadão mesmo pelo comentário. Ajudou muito!

**Comment by luizangelomedeiros** [comment]
@camillodev Que bom cara , fico feliz 😄 
nao podia deixar de contribuir com minha experiência, pois tbm tive pessoas que me ajudaram quando eu estava nesse processo [(203)](https://github.com/frontendbr/forum/issues/203) e foi muito útil. 

Se decidir vir para floripa, me avisa, que dae posso passar mais algumas dicas específicas daqui. 
boa sorte 👊

**Comment by ValentinaDias** [comment]
Gostaria de recomendar o site da [Jooble](https://br.jooble.org) é o site de procura de emprego no Brasil e no exterior,  ele me ajudou bastante durante a procura das vagas no RJ. Como sou estrangeira não foi fácil achar um trabalho num país desconhecido, mas este site gerou as vagas dos fontes diferentes (vários sites, mais populares no Brasil. Eu não presicava passar por todas as sites separadamente.

Tomara que ajude!

**Comment by JulianoPadilha** [comment]
Eu era de Belém/PA e me mudei para SP há uns 8 meses. Eu vim na cara e na coragem e as coisas têm dado certo. :D  

Eu estava no último ano da faculdade e sabia que quando me formasse não queria continuar em Belém, então fiz o planejamento do que eu precisava para mudar para SP assim que me graduasse. A principal dificuldade, obviamente, é a financeira. Então consegui um emprego com um professor onde mesmo em Belém eu já trabalhava remoto. Nesse período eu passei a aplicar pra vagas em SP, mas as empresas sempre retornavam dizendo que por eu ainda não ser do estado isso dificultava o processo. =/

Eu sabia que se eu fosse esperar conseguir um emprego eu nunca me mudaria, então juntei uma grana (suficiente para ficar 2 meses em SP) e me mudei assim que me formei. Tive a sorte nessa história toda de ter a ajuda do meu professor, que me deu a oportunidade de trabalhar remoto ainda em Belém e me manteve remoto quando eu já estava em SP e não tinha um emprego. Após 2 meses em SP consegui um trabalho e agora estou feliz. rs Fim.

Mesmo se eu não tivesse contado com a força do meu professor, eu ainda assim faria tudo de novo e me mudaria. Vim pra cá sem lugar pra ficar, sem conhecer ninguém e sem emprego. Isso me forçou a ser criativo com o que tinha e amadureci durante a jornada. Na dúvida, sempre faça o que te dá mais medo, pois a recompensa será maior. \o

Boa sorte.

**Comment by munizart** [comment]
Opa @camilodev,
Ainda não vivi essa experiência na minha carreira, mas quando mais novo viva me mudando por conta da carreirado do meu pai. Então vou dividir aqui um pouco do que eu peguei do processo dele.
 - Quando ele achava uma oportunidade (ou a oportunidade achava ele) ele se aplicava pra vaga não dependendo a cidade e tentava sempre participar dos processos seletivos pessoalmente.
 - Ele só se mudava depois de ter a vaga garantida (imagino que por conta dos filhos)
 - Ele sempre consegui negociar os custeios das mudanças com o empregador

Apesar de ele não ser da area de front-end sempre vi isso como bem positivo pra carreira dele, já que ele conseguiu uma progressão de conhecimento e remuneração maior do que se ficasse no mesmo lugar.
Pretendo seguir esse caminho assim que me graduar.

Espero que ajude =]

Ah, e uma dica boa é deixar já no começo do processo seletivo claro que és de fora e tens a intenção de se mudar. Se o empregador ver isso como impeditivo, mais rapido você parte pra próxima oportunidade.
Boa sorte

**Comment by ghost** [comment]
@camillodev ótima discussão você abriu aqui, normalmente sempre vemos artigos como é o processo de sair do Brasil e pouco se fala como é sair do seu estado.

Sou cearense de uma cidade chamada Maracanaú que é um dos municipios que fazem parte da  região metropolitana de Fortaleza. Desde meu inicio na carreira passei 4 anos só no Ceará. Posteriormente tive a oportunidade de vir para São Paulo onde estou há 2 anos.

A primeira coisa que aprendi ao chegar em SP é que a comunidade de desenvolvimento no geral é bem unida. Apesar das "picuinhas de linguagens e bullying" saiba que você vai ter muito apoio de todos para aprender e evoluir.

A comunidade aqui realmente é muito ativa, e foi fácil enturmar-se.

O Mercado de trabalho é fantástico, tem muita vaga de diferentes níveis e especialidades então saiba que vagas existem mas será dificil. Não desista e nem fique chateado se não passar em alguma entrevista. Foco na vaga e vai pra cima hahhaaha

Para achar vagas sempre sugiro conferir o Lovemondays e fazer um wishlist de algumas empresas que você acha que seria dahora trabalhar.

Por fim, moradia normalmente é caro dependendo de onde você venha morar, mas uma dica: Melhor pagar um pouco caro pra morar perto do trabalho do que passar muitas horas em ônibus e metrôs . Vai por mim tu pode aproveitar esse tempo em uma Faculdade, Curso, Meetups enfim muita coisa.
